daniel-aguirre-salamanca/tarea-1/magazine.html

```diff
@@ -8,43 +8,41 @@
     <title>MAGAZINE</title>
   </head>
   <body>
-    <section class="container">
-      <div class="magazine">
-        <div class="magazine__header">
-          <span>THE</span>
-          <span>BUSINESS</span>
-          <span>MAGAZINE</span>
-        </div>
-        <div class="magazine__content">
-          <div class="magazine__ideas">
-            <div class="magazine__idea">
-              <div class="idea__title">MAKE YOUR <br/>NEW IDEA</div>
-              <div class="line-breaker"></div>
-              Lorem ipsum dolor sit amet consectetur adipisicing elit.
-            </div>
-            <div class="magazine__idea">
-              <div class="idea__title">MEET THE <br />REAL LEADER</div>
-              <div class="line-breaker"></div>
-              Lorem ipsum dolor sit amet consectetur adipisicing elit.
-            </div>
-            <div class="magazine__idea">
-              <div class="idea__title">STEP UP <br/> YOUR BUSINESS</div>
-              <div class="line-breaker"></div>
-              Lorem ipsum dolor sit amet consectetur adipisicing elit.
-            </div>
-          </div>
-          <div class="magazine__main-title">
-            YOUR WAY <br />TO SUCCESS
-            <div class="magazine__main-text">
-              Lorem ipsum, dolor sit amet consectetur <br/>adipisicing elit.
-            </div>
-          </div>
-          <div class="qr">
-              ISSUE 02 - MAY 2028
+    <main class="container">
+      <article class="magazine">
+        <header class="magazine__header">
+          <h2>THE</h2>
+          <h1>BUSINESS</h1>
+          <h2>MAGAZINE</h2>
+        </header>
+        <section class="magazine__content">
+          <ul class="magazine__ideas">
+            <li class="magazine__idea">
+              <h3>MAKE YOUR <br/>NEW IDEA</h3>
+              <hr/>
+              <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+            </li>
+            <li class="magazine__idea">
+              <h3>MEET THE <br />REAL LEADER</h3>
+              <hr/>
+              <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+            </li>
+            <li class="magazine__idea">
+              <h3>STEP UP <br/> YOUR BUSINESS</h3>
+              <hr/>
+              <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+            </li>
+          </ul>
+          <article class="magazine__main">
```

Comment:
@daguirreglobant , this one should be a section

Reply:
@emolinaglobant Done!